### PR TITLE
Update website with some stylez and some new information

### PR DESCRIPTION
**Pull request by darcyclarke**
Check it out and let me know your thoughts... :D would like at least two :+1:'s before merging 

/cc @jeffjewiss @rickycodes @Rchristiani @brenna @andrewdamelio @richgilbank @ahmadnassri

### CODE REVIEW

**Comment by ahmadnassri** [comment]
lemme run this locally ...

**Comment by ahmadnassri** [comment]
1. whoa! holy redesign batman!
2. terminal looks broken (on a narrow window):
   image
3. hovering over the background gallery seems clunky, messes with the `z-index` of the animation on top of the mentor avatars

**Comment by ahmadnassri** [comment]
e.g. of `3`

image

**Comment by ahmadnassri** [comment]
seems to look for sockets ..
image

**Comment by jeffjewiss** [comment]
Looks slick, thanks for putting so much work into this!

Are we ditching the colour scheme from the stickers? I thought some of us had discussed using a more unique set of colours for Toronto instead of the NodeSchool black and yellow, but I can't remember.

I like the visual style the icons add, but on a first viewing I don't know what any of them mean and my assumption is that most visitors are on the site to quickly find information about the event and likely not return visitors. Do you think it would be clearer if those links had text as well as icons?

**Comment by jeffjewiss** [comment]
@ahmadnassri I think the sockets stuff is for the Slack badge. Maybe it doesn't work locally.

File: styles.sass [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+
+.tweets
+
+  iframe, iframe[id*="twitter-widget"], .timeline
+    max-width: 100%
+    width: 100%
+
+  @media (screen and max-width: 800px)
+    display: none
+  s
```
Comment (jeffjewiss) [inline styles.sass]: Is this a typo? :stuck_out_tongue:

**Comment by darcyclarke** [comment]
It must be the particle js because I didn't add anything.... also lol it seems to be happening on the current website: http://nodeschool.io/toronto/ So I blame @jeffjewiss :P

File: styles.sass (hunk 1 above)
Comment (darcyclarke) [inline styles.sass]: NO! Yessum

**Comment by darcyclarke** [comment]
@jeffjewiss I'd be down to use the "NodeSchool Toronto Red" a bit more... I wanted to go a bit more minimal with the colors and then ended up landing back more towards white/black and then the default NodeSchool yellow seemed to fit. I'm also down to just loose the icons/nav if it doesn't add anything. I agree that this website should act as a quick reference for information.

I'm also open to adding text too... maybe on hover? Iunno... I'm easy.

**Comment by jeffjewiss** [comment]
Re: sockets, there's a bug filed on the slackin' repo, so I think it might be out of our control.

I'd like a few other people to weigh in. This version has a lot more info than the current website, so colours shouldn't matter too much.

**Comment by Fishrock123** [comment]
Is it possible to use GitHub's Large File Storage for the images? I don't remember if that is paid or not for things like this.

Also lmk if you want a picture of me that isn't a fish, though I don't mind being a fish.

**Comment by Fishrock123** [comment]
Oh and, what's this look like on mobile?

**Comment by darcyclarke** [comment]
@Fishrock123 could you shoot me a non-fish pic of yourself?

**Comment by vladikoff** [comment]
Awesome work Darcy!! It is alive ! http://nodeschool.io/toronto/

@darcyclarke the only thing that I'm wondering if it is clear enough that the " > " is clickable and is the way to get tickets or check the waiting list. What do you think? The animation that you have right now might be enough

image

**Comment by darcyclarke** [comment]
@vladikoff hmm yea... I agree. Maybe it should be more explicit messaging (ie. "Register for our next event: Saturday, Oct 24th at Shopify"???)

**Comment by vladikoff** [comment]
@darcyclarke that works, would separating the button and the map help or an underline for "Register" ?

**Comment by darcyclarke** [comment]
@vladikoff 
screen shot 2015-10-20 at 3 07 06 pm Thoughts?

**Comment by vladikoff** [comment]
Nice! can also be "**Click to check ticket availability** for your next event on", but "Click to register" is also great!

@rickycodes IDEAS? :bulb:

**Pull request merged**